proprietary/Configure-TI-CC2541-compatible-proprietary-PHY.md

```diff
@@ -0,0 +1,134 @@
+<details>
+<summary><font size=5>Table of Contents</font> </summary>
+
+- [1. Introduction](#1-introduction)
+- [2. Prerequiesite](#2-Prerequisite)
+- [3. PHY Confirm](#3-PHY-Confirm)
+- [4. EFR32xG22 project](#4-EFR32xG22-project)
+- [5. Conclusion](#5-Conclusion)
+
+</details>
+
+# 1. Introduction
+Sometime we got request on configure compatiable PHY for communicate with device from other vendor. Here share the experiece on configure an EFR32xG22 PHY communicate with TI CC2541 [proprietary 2.4G PHY](files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/2M_BS-TX_2.xml). 
+To configure the PHY, we need to know the detail parameter on radio configurator.
+
+# 2. Prerequisite 
+
+## 2.1. Hardware Requirement
+* 1 [WSTK with EFR32MG22 radio boards(BRD4182A)](https://www.silabs.com/development-tools/wireless/efr32xg22-wireless-starter-kit)
+* 1 SmartRF05 EB plus [CC2541EMK](https://www.ti.com/tool/CC2541EMK)
+* 1 [Anritsu MS2692A](https://www.anritsu.com/en-US/test-measurement/products/ms2692a)
+
+The BRD4182A radio board supports three wireless protocols. Bluetooth LE/Mesh, Zigbee and Proprietary.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/wstk.png">  
+</div> 
+CC2541EMK is the one to communicate with, we need it for confirm the communication. And confrim some unknown feature from its air data.
```

Comment:
CC2541EMK supports the Proprietary to communicate with BRD4182A radio board, it is needed to establish the communication. And get some unknown features from its air data.